api/feedback.go

```diff
@@ -43,24 +46,27 @@ func GenerateFeedbackMessage(f *models.Feedback, from, to string) []byte {
 
 	b.WriteString(fmt.Sprintf("From: %s\n", from))
 	b.WriteString(fmt.Sprintf("To: %s\n", to))
-	b.WriteString(fmt.Sprintf("Subject: Feedback received\n\n"))
+	b.WriteString("Subject: Feedback received\n\n")
 
-	b.WriteString(fmt.Sprintf("Is page useful: %t\n", *f.IsPageUseful))
-	b.WriteString(fmt.Sprintf("Is general feedback: %t\n", *f.IsGeneralFeedback))
+	if *f.IsPageUseful {
+		b.WriteString(fmt.Sprintf("Feedback Type: %s\n", mapper.ASpecificPage))
```

Comment:
```suggestion
	if !*f.IsGeneralFeedback {
		b.WriteString(fmt.Sprintf("Feedback Type: %s\n", mapper.ASpecificPage))
```

Reply:
or you could even just provide the string based off that, saves a minor point of duplication.